.github/workflows/coverage-check.yml

```diff
@@ -49,7 +49,15 @@ jobs:
               --rc lcov_branch_coverage=1 \
               --remove merged-lcov.info \
               --output-file filtered-lcov.info \
-              "*node_modules*" "*test*" "*mock*" "*scripts*"
+                  "*node_modules*" \
+                  "*test*" \
+                  "*mock*" \
+                  "*scripts*" \
+                  "src/dollar/libraries/*" \
```

Comment:
In the original issue the `src/dollar/libs/*` path is mentioned to be removed from coverage (where we used to store mocked libs). The `src/dollar/libraries/*` path actually contains facet libs which should be tested. Could you remove this `src/dollar/libraries/*` path from coverage filters?

Reply:
ok, updated it